go/mars/register_controllers.gen.go

```diff
@@ -0,0 +1,37 @@
+// DO NOT EDIT -- code generated by mars-gen
+package main
```

Comment:
could this file be generated on the container (before compilation) ?

Reply:
Yes, just add `go generate main.go` before running the build.

Reply:
may I push a fix on your branch ?

Reply:
Yes, sure!